Source/WebCore/animation/KeyframeEffect.cpp

```diff
@@ -2894,7 +2894,12 @@ void KeyframeEffect::effectStackNoLongerAllowsAccelerationDuringAcceleratedActio
 {
 #if ENABLE(THREADED_ANIMATIONS)
     if (canHaveAcceleratedRepresentation()) {
-        ASSERT_NOT_REACHED();
+        if (RefPtr document = this->document()) {
```

Comment:
How about putting this code in a lambda that's only called from the assertion, to remove the overhead of getting `document()` and `settings()` in release.